src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -0,0 +1,151 @@
+
+    with self.argument_context('ams transform list') as c:
+        c.argument('account_name', id_part=None)
+
+    with self.argument_context('ams asset') as c:
+        c.argument('account_name', account_name_arg_type, id_part='name',
+                   help='The name of the Azure Media Services account within the resource group.')
+        c.argument('asset_name', name_arg_type, id_part='child_name_1',
+                   help='The name of the asset.')
+
+    with self.argument_context('ams asset list') as c:
+        c.argument('account_name', id_part=None)
+
+    with self.argument_context('ams asset create') as c:
+        c.argument('alternate_id', help='The alternate id of the asset.')
+        c.argument('description', help='The asset description.')
+        c.argument('asset_name', name_arg_type, help='The name of the asset.')
+
+    with self.argument_context('ams asset get-sas-urls') as c:
+        c.argument('permissions', arg_type=get_enum_type(AssetContainerPermission),
+                   help='The permissions to set on the SAS URL.')
+        c.argument('expiry_time', expiry_arg_type, help="Specifies the UTC datetime (Y-m-d'T'H:M'Z') at which the SAS becomes invalid.")
+
+    with self.argument_context('ams job') as c:
+        c.argument('account_name', account_name_arg_type, id_part='name',
+                   help='The name of the Azure Media Services account within the resource group.')
+        c.argument('transform_name', transform_name_arg_type, id_part='child_name_1',
+                   help='The name of the transform.')
+        c.argument('job_name', name_arg_type, id_part='child_name_2',
+                   help='The name of the job.')
+
+    with self.argument_context('ams job list') as c:
+        c.argument('account_name', id_part=None)
+
+    with self.argument_context('ams job start') as c:
+        c.argument('priority', arg_type=get_enum_type(Priority),
+                   help='The priority with which the job should be processed.')
+        c.argument('description', help='The job description.')
+        c.argument('input_asset_name',
+                   arg_group='Asset Job Input',
+                   help='The name of the input asset.')
+        c.argument('output_asset_names',
+                   nargs='+', help='Space-separated list of output asset names.')
+        c.argument('base_uri',
+                   arg_group='Http Job Input',
+                   help='Base uri for http job input. It will be concatenated with provided file names. If no base uri is given, then the provided file list is assumed to be fully qualified uris.')
+        c.argument('files',
+                   nargs='+',
+                   help='Space-separated list of files. It can be used to tell the service to only use the files specified from the input asset.')
+
+    with self.argument_context('ams job cancel') as c:
+        c.argument('delete', help='Delete the job being cancelled.')
```

Comment:
Also needs to be registered as a flag. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags

Reply:
Done